src/components/clusterConfiguration/discoveryTroubleshootingModal.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import {
+  Button,
+  Modal,
+  ButtonVariant,
+  ModalVariant,
+  TextContent,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
+import { ToolbarButton } from '../ui/Toolbar';
+import PrismCode from '../ui/PrismCode';
+
+type DiscoveryTroubleshootingModalButtonProps = React.ComponentProps<typeof Button> & {
+  ButtonComponent?: typeof Button | typeof ToolbarButton;
+  onClick?: () => void;
+};
+
+export const DiscoveryTroubleshootingModalButton: React.FC<DiscoveryTroubleshootingModalButtonProps> = ({
+  ButtonComponent = Button,
+  onClick,
+  children,
+  ...props
+}) => {
+  const [isModalOpen, setIsModalOpen] = React.useState(false);
+  const closeModal = () => setIsModalOpen(false);
+  const handleClick = onClick || (() => setIsModalOpen(true));
+  return (
+    <>
+      <ButtonComponent variant={ButtonVariant.link} {...props} onClick={handleClick} isInline>
+        {children}
+      </ButtonComponent>
+      <DiscoveryTroubleshootingModal isOpen={isModalOpen} onClose={closeModal} />
+    </>
+  );
+};
+
+type DiscoveryTroubleshootingModalProps = {
+  onClose: () => void;
+  isOpen: boolean;
+};
+
+export const DiscoveryTroubleshootingModal: React.FC<DiscoveryTroubleshootingModalProps> = ({
+  onClose,
+  isOpen,
+}) => {
+  return (
+    <Modal
+      title="Host Discovery Troubleshooting"
+      isOpen={isOpen}
+      actions={[
+        <Button key="close" variant={ButtonVariant.primary} onClick={onClose}>
+          Close
+        </Button>,
+      ]}
+      onClose={onClose}
+      variant={ModalVariant.large}
+    >
+      <TextContent>
+        <Text component={TextVariants.p}>
+          Booting up an ISO might take a few minutes, and might depend heavily on your hardware and
+          network configuration, however, if after a few minutes your machine does not show up in
+          the UI, it's probably best to try and login to it via SSH, for that you would need to know
+          the machine ip-address, or use a console of some sort.
+        </Text>
+        <Text component={TextVariants.h2}>SSH into your machine</Text>
+        <PrismCode code={`ssh core@<machine-ip>`} />
+        <Text component={TextVariants.p}>
+          If SSH key was provided when generating the ISO it is not required to provide a password.
```

Comment:
do they even know the password? I thought only SSH key base auth will work

Reply:
I've reworded the section and made the SSH key in discovery iso dialog mandatory to ensure users ability to debug the registration